docs/source/hotregion.rst

```diff
@@ -8,6 +8,20 @@ HotRegion
 Instances of :class:`~.HotRegion.HotRegion` are objects representing radiatively
 intense regions of the source photosphere.
 
+.. figure:: images/super_cede_omit.png
+   :width: 600
+
+Representation of the class instantiation options for the parameters `omit` and `cede` described below. 
```

Comment:
Representation of the ``HotRegion`` class instantiation options ``omit`` and ``cede``. The image also shows the prefixes used for the instance attributes associated with each type of region constituting a ``HotRegion`` object.

Reply:
hi @serenavinciguerra89 , I made some edits to your caption. Can you please check and push the edits? Thanks!